ci/terraform/main.tf

```diff
@@ -37,18 +146,14 @@ resource "google_cloudbuild_trigger" "integ-tests-on-push" {
     owner = var.repo_owner
     name  = var.repo_name
     push {
-      branch = var.repo_branch
+      branch = var.integ_tests_branch
     }
   }
 
   substitutions = {
-    _RELEASE_BUCKET  = google_storage_bucket.build-artifacts.name
+    _RELEASE_BUCKET  = google_storage_bucket.build_artifacts.name
     _RELEASE_CHANNEL = "${var.release_channels[count.index]}"
   }
 
-  # Avoid wasting resources each time the PR is updated by 
-  # only running builds that are explicitly approved.
-  approval_config {
```

Comment:
will this build still require approval for unknown contributors?

Reply:
I decided to instead restrict runs to a post submit (i.e. only for changes on 'main'). If we want to trigger integ tests as presubmits the triggers can be executed through the CloudBuild UI.

The build approval feature doesn't seem to be well supported by the github integration (you need to go to cloudbuild and manually approve the builds regardless AND the links from the github UI to workflows requiring approval don't actually work correctly so you have to search for the build).